frontend/packages/dev-console/src/components/taskruns/TaskRunDetails.tsx

```diff
@@ -15,6 +19,15 @@ const TaskRunDetails: React.FC<TaskRunDetailsProps> = ({ obj: taskRun }) => {
         <div className="col-sm-6">
           <ResourceSummary resource={taskRun} />
         </div>
+        <div className="col-sm-6 odc-taskrun-details__status">
+          <dl>
+            <dt>Status</dt>
+            <dd>
+              <Status status={pipelineRunFilterReducer(taskRun)} />
+            </dd>
+          </dl>
+          <PipelineRunDetailsErrorLog pipelineRun={taskRun} />
```

Comment:
These two pipelineRun specific functions called in the context of `taskRun` looks odd to me. If taskRun has more specific type definition, then this will cause lint errors and confusions in future.  It will be good to refactor these to a generic function calls. WDYT?

Reply:
I've updated such that PLR and TR uses separate utils and point to a common component